cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -260,3 +289,37 @@ def test_custom_button_automate(request, appliance, context, submit, objects, bu
                 assert False, "Expected {count} requests not found in automation log".format(
                     count=str(expected_count)
                 )
+
+
+@pytest.mark.meta(blockers=[BZ(1659452, forced_streams=["5.9", "5.10"],unblock=lambda button_group: "group" not in button_group)])
+@pytest.mark.parametrize("context", [ViaUI, ViaSSUI])
+def test_custom_button_text_display(appliance, context, button_group, service):
+    """ Test custom button text display on option
+
+    Bugzilla:
+        * 1650066
+
+    Polarion:
+        assignee: ndhandre
+        caseimportance: medium
+        initialEstimate: 1/6h
+        testSteps:
+            Prerequisites:
+                * Appliance with Service
+            Steps:
+                * Create custom button `Group` or `Button` without display option
+                * Check Group/Button text display or not on UI and SSUI.
+    """
+
+    my_service = MyService(appliance, name=service.name)
+    button, group = button_group
```

Comment:
These are backwards from what is yielded in the fixture.

Reply:
my bad; it was creating fixture conflict and bad use. corrected